templates/primary-secondary/primary-secondary.js

```diff
@@ -112,15 +113,18 @@ class DesktopKeyboardResizer extends Resizer {
 		if (this.isMobile) {
 			return;
 		}
-		const leftKeyCode = this.isRtl ? keyCodes.RIGHT : keyCodes.LEFT;
-		const rightKeyCode = this.isRtl ? keyCodes.LEFT : keyCodes.RIGHT;
-		if (e.keyCode !== leftKeyCode && e.keyCode !== rightKeyCode) {
+		if (e.keyCode !== keyCodes.LEFT && e.keyCode !== keyCodes.RIGHT) {
 			return;
 		}
 		e.preventDefault();
+
+		// The direction of the container is flipped if exactly one of isRtl and secondaryFirst is true
+		const isFlipped = this.isRtl ^ this.secondaryFirst;
+		const direction = e.keyCode === keyCodes.LEFT && !isFlipped
+			|| e.keyCode === keyCodes.RIGHT && isFlipped ? 1 : -1;
```

Comment:
The resizer works by adjusting the width of the secondary container. If the page is in RTL then the placement of the secondary container is flipped but if `secondaryFirst` is also active then it will flip back to the original position. So the direction will be flipped when exactly one of `isRtl` and `secondaryFirst` is set to true.